schema/me/notifications.js

```diff
@@ -46,6 +46,10 @@ const NotificationsFeedItemType = new GraphQLObjectType({
         },
       }),
     },
+    image: {
+      type: GraphQLString,
+      resolve: ({ object }) => object.artist.image_url,
```

Comment:
Up to you, I'm not a great namer. If you think `image` for a feed item is confusing you can be more specific, but `image` _could_ be generic enough to just be: 'the image that represents this feed item'. Currently it's an artist avatar, in the future it might be something else (depending on feed item type perhaps), whereas the client could perhaps just always be rendering `image. That may make it easier to support more complex feeds in the future.

So, it could be nice to have had all that encapsulated in the more generic `image`.

Actually, I think I convinced myself while typing that maybe `image` _is_ the most appropriate name.